opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -179,6 +185,21 @@ def __init__(
         self._shutdown_once = Once()
         self._shutdown = False
 
+        self._views = views
```

Comment:
We don't need this on the class and in sdk_config. Can we set it once on the sdk_config and use that everywhere?

Reply:
Fixed